UI_UX/Frontend/src/styles/AppHeader.scss

```diff
@@ -1,21 +1,81 @@
-$headerHeight: 74px;
+$headerHeight: 90px;
 
 .AppHeader {
 	height: $headerHeight;
-	background-color: #200344;
+	background-image: linear-gradient(to right, #0b3f9c, #2e58db);;
     margin: 0px 0px;
     color: white;
 
     .AppHeader_logo {
         padding: 0 25px;
         margin: auto 0;
-        height: 47px;
+        height: 73px;
```

Comment:
We are fine with magic numbers for now, eventually we will use a `spacings.scss` and import them as variables.